lib/routes/sign.js

```diff
@@ -2,6 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+var crypto = require('crypto')
+
+var TIMED_KEY = crypto.randomBytes(8)
+setInterval(function () {
+  TIMED_KEY = crypto.randomBytes(8)
+}, 1000 * 60 * 60 * 24);
```

Comment:
Yea, I thought of this timed key before I finally found where to log the info, and I didn't know the uid would be logged also. This was to prevent trying to find the uid based on the patterns over time. I'll just kill it.